app/src/CoursePages/CoursePage.css

```diff
@@ -7,17 +7,141 @@
   right: 0%;
   margin: 3%;
   gap: 1rem;
+  overflow: scroll;
+  justify-content: space-around;
+}
+
+.ratingbox {
+  font-weight: bold;
+  font-size: clamp(1px, 130%, 3vw);
+  display: flex;
+  align-items: center;
+  align-self: flex-end;
+  justify-content: center;
+  height: 15vh;
+  max-height: 4ch;
+  min-width: 3ch;
+  width: 8vw;
+}
+
+.ratingbox-Difficulty {
+  background-color: #e087e2;
+}
+
+.ratingbox-Workload {
+  background-color: #8dbbff;
+}
+
+.ratingbox-Practicality {
+  background-color: #88ecaa;
 }
 
+/*
+  CSS styling for the top portion of the course page
+*/
+
 .topbox {
+  display: flex;
+  background-color: #d3f5ff;
+  padding: 2rem;
+  align-items: center;
+  justify-content: space-between;
+}
+
+/*
+  Left portion of the top box containing the course title, description
+  and buttons
+*/
+.leftflex {
+  display: flex;
+  flex-direction: column;
+}
+
+/*
+  Right portion of the top box containing the overall ratings
+*/
+.rightflex {
   display: flex;
   flex-direction: column;
-  height: 20rem;
   text-align: center;
-  background-color: #d3f5ff;
+  width: 40%;
+}
+
+/*
+  Flex-box for the course number and name
+*/
+.coursetitle {
+  display: flex;
+  align-items: center;
+  margin-bottom: 1rem;
+  gap: 1rem;
+}
+
+/*
+  Number for the course (e.g. CSE 351)
+*/
+.coursenum {
+  text-wrap: nowrap;
+  margin: 0;
+
+}
+
+/*
+  The name of the course (e.g. "Hardware/Software Interface")
+*/
+.coursename {
+  text-wrap: balance;
+  margin: 0;
+}
+
+.coursedesc {
+  text-align: left;
+  max-width: 30rem;
+  height: auto;
+  text-wrap: balance;
+}
+
+.buttons-flexbox {
+  display: flex;
+  align-items: flex-start;
+  gap: 1rem;
+  margin-top: 1rem;
 }
 
-/* Reviews section */
+.purplebutton {
+  background-color: #B175FF;
+  color: white;
+  box-shadow: 3px 4px 4px 0px rgba(255, 255, 255, 0.35) inset;
+  padding: 1.8%;
+  border: none;
+  outline: none;
+  font-size: 1.04rem;
+  font-weight: 700;
+}
+
+.purplebutton:hover {
+  background-color: #8B53D3;
+  cursor: pointer;
+}
+
+.overallratings-header {
+  font-size: 1.75rem;
+  margin: 0 0 2.5rem 0;
+}
+
+/*
+  Flexbox for the rating boxes in the "Overall Ratings" section
+*/
+.ratings-flexbox {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+}
+
+/*
+  CSS styling for the bottom portion of the course page (the reviews)
+  starts here.
```

Comment:
Maybe remove "starts here". That is implied based on how our other comments are like.

Reply:
Fixed.